### PR TITLE
Add Prowlarr to Easypanel 🎉

### PR DESCRIPTION
**Pull request by Supernova3339**
Don't look at the commit title

### CODE REVIEW

File: templates/prowlarr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      type: string
+      title: Project Name
+    appServiceName:
+      type: string
+      title: App Service Name
+      default: prowlarr
+    appServiceImage:
+      type: string
+      title: App Service Image
+      default: linuxserver/prowlarr
```
Comment (deiucanta) [inline templates/prowlarr/meta.yaml:34]: Can we use a fixed version?

**Comment by deiucanta** [comment]
Please remove the lock files and set a fixed version for the image.

**Comment by Supernova3339** [comment]
> Please remove the lock files and set a fixed version for the image.

All that is available is dev and nightly builds. Which branch would you suggest?

**Comment by deiucanta** [comment]
Merged locally

**Pull request closed without merging**